### ISSUE TITLE
main-page-icons

### ISSUE THREAD

**Issue by mar-bi** [chingu-voyage3/grad.then#53]
**Icons** for the **Main Page** 
---
This is how the icons are used on the `Main` Page
![iconsmainpage](https://user-images.githubusercontent.com/25195616/34034096-bd30fe90-e185-11e7-9174-5b8f9fd43ec2.png)
---
Let's look at what I picked up for `Main` Page.  
1. Icons for `Jobs`
![jobs](https://user-images.githubusercontent.com/25195616/34035834-efd3d484-e18b-11e7-9912-bf6398168033.png)

2. Icons for `Learn`
![learn](https://user-images.githubusercontent.com/25195616/34035919-3f58d34c-e18c-11e7-8636-c16c012c5c67.png)

3. Icons for `Events` (other keyword - **community**)

![events](https://user-images.githubusercontent.com/25195616/34036376-ad512876-e18d-11e7-8109-144090510a54.png)

4. Icons for `Support` (other keyword - **contribute**)

![support](https://user-images.githubusercontent.com/25195616/34036308-7485c97a-e18d-11e7-8ee6-b82c60d3330f.png)

5.  Icons for `Stories` (other keyword - **share**)

![stories](https://user-images.githubusercontent.com/25195616/34036329-84ac8244-e18d-11e7-915f-34bba0396f21.png)

All these icons are in SVG format.

**Please add your comments which icons you'd like to choose.**

You can offer other icons for any of these meaningful groups. Just be sure that you can provide its `path`. It has this form: 
```
<path d="M10 4h8v-2h-8v2zM28 14v7.5c0 1.375-1.125 2.5-2.5 2.5h-23c-1.375 
0-2.5-1.125-2.5-2.5v-7.5h10.5v2.5c0 0.547 0.453 1 1 1h5c0.547 0 1-0.453 
1-1v-2.5h10.5zM16 14v2h-4v-2h4zM28 6.5v6h-28v-6c0-1.375 
1.125-2.5 2.5-2.5h5.5v-2.5c0-0.828 0.672-1.5 1.5-1.5h9c0.828 
0 1.5 0.672 1.5 1.5v2.5h5.5c1.375 0 2.5 1.125 2.5 2.5z" />
```

**Comment by mar-bi** [comment]
I prefer next icons:
1. Any briefcase, but **briefcase-3** more
2. **icon-books**
3. last 2 icons in the row - **group-2** and **slideshare**
4. first 3 icons - **ruler** and both **puzzles**
5. **icon-blog**

**Comment by spences10** [comment]
Same here, I'll go with your choice @mar-bi 

The only ones I was concerned about was `Jobs`, looks like a toolbox, but hey that's fine, and I prefer it to the other two, so moot point 🙃 and the `Events` one I preferred the look of icon-group-2 but thinking about it your selection makes more sense.

**Comment by mar-bi** [comment]
What about **briefcase-2**, @spences10 ?
It has 2 nice locks. :upside_down_face: 
For `events` you like **icon-group-2**, right?

**Comment by mar-bi** [comment]
And what is about `support-contribute`? 
**Puzzles** or **Pencil&Ruler** ?

**Comment by spences10** [comment]
**briefcase-2** 👍 
`Events` **icon-group-2** Yes
**Pencil&Ruler** 👍

**Comment by mar-bi** [comment]
Cool!😎
I can start building icons 🎉💻🔧🔨

**Comment by mar-bi** [comment]
@asanhix and @miljan-fsd 
you can add your voices. The discussion is still open. 
I've started to build **icons**, and it doesn't cost anything to switch from one icon to another. :upside_down_face:

**Comment by asanhix** [comment]
@mar-bi thanks All fine with me if something is off with the design we  can modify it later right?

**Comment by mar-bi** [comment]
yes, exactly. 
we can modify it later :smile:

**Comment by mar-bi** [comment]
added via `Icon.js` and `constants.js`.
closed with #61
